packages/flutter/lib/src/services/raw_keyboard_linux.dart

```diff
@@ -39,32 +39,31 @@ class RawKeyEventDataLinux extends RawKeyEventData {
   /// (GLFW, GTK, QT, etc) may have a different key code mapping.
   final KeyHelper keyHelper;
 
+  /// The name of the printable key, encoded as UTF-8. This is
+  /// typically the character that key would produce without any modifier keys.
+  /// For dead keys, it is typically the diacritic it would add to a character.
+  /// Defaults to an empty string, asserted to be not null.
+  final String codePoint;
```

Comment:
this field says `String`, which in Dart is always UTF-16, but the docs say "encoded as UTF-8".

Reply:
Sorry, copied that from the GLFW documentation. Fixed the comment.